src/python/grpcio/grpc/_channel.py

```diff
@@ -99,6 +99,22 @@ def _wait_once_until(condition, until):
     else:
       condition.wait(timeout=remaining)
 
+_INTERNAL_CALLERROR_MESSAGE = (
```

Comment:
Since this contains a `%` to be replaced it is a message format rather than a message. `_INTERNAL_CALL_ERROR_MESSAGE_FORMAT`?